lib/ansible/plugins/filter/vault.yml

```diff
@@ -29,18 +29,41 @@ DOCUMENTATION:
         - This toggle can force the return of an C(AnsibleVaultEncryptedUnicode) string object, when V(False), you get a simple string.
         - Mostly useful when combining with the C(to_yaml) filter to output the 'inline vault' format.
       type: bool
-      default: False
+      default: false
 
 EXAMPLES: |
-  # simply encrypt my key in a vault
+  # Example 1: Encrypt a variable using the vault filter
   vars:
-    myvaultedkey: "{{ keyrawdata|vault(passphrase) }} "
+    keyrawdata: "my_secret_key"
+    passphrase: "my_vault_password"
+    myvaultedkey: "{{ keyrawdata | vault(passphrase) }}"
 
-  - name: save templated vaulted data
-    template: src=dump_template_data.j2 dest=/some/key/vault.txt
-    vars:
-      mysalt: '{{2**256|random(seed=inventory_hostname)}}'
-      template_data: '{{ secretdata|vault(vaultsecret, salt=mysalt) }}'
+  # Example 2: Encrypt a value and save it to a file using the template module
```

Comment:
I would avoid the `# comments` as they tend to get stripped in the HTML versions

Reply:
yep i removed the most of them just kept a few to elaborate the exmaple, let me know what you think!